README.md

```diff
@@ -42,10 +44,6 @@ Before applying the configuration, please update the image path in the YAML file
 kubectl apply filename=deploy/deploy.yaml
```

Comment:
Just a note that we should refer to our Helm chart once it's available.

Reply:
I have this in mind. But thanks for being thorough :)